### PR TITLE
chore(build): Update github action versions

### PR DESCRIPTION
**Pull request by rasa**
Built [fine](https://github.com/rasa/HumanizeDuration.js/actions/runs/9148920943) in my fork.

### CODE REVIEW

**Comment by rasa** [comment]
I also tested on the beta ubuntu-24.04, and it ran fine.

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
         with:
-          node-version: 20
+          node-version: 22
```
Comment (EvanHahn) [inline .github/workflows/nodejs.yml:13]: Could you revert this, or test on both v20 and v22?

File: .github/workflows/nodejs.yml [hunk 2]
```diff
@@ -1,15 +1,15 @@
 name: Node.js CI
 
-on: [push, pull_request]
+on: [push, pull_request, workflow_dispatch]
```
Comment (EvanHahn) [inline .github/workflows/nodejs.yml:3]: Could you revert this?

**Review by EvanHahn — CHANGES_REQUESTED** [review]
Looks good other than my small comments. Thanks for doing this!

File: .github/workflows/nodejs.yml (hunk 1 above)
Comment (rasa) [reply .github/workflows/nodejs.yml:13]: Sure, will test on 20

**Comment by rasa** [comment]
@EvanHahn done. 22 has only been out three weeks, but appears stable.

**Review by EvanHahn — APPROVED** [review]
Thank you!

**Pull request merged**